Runtime/Patterns/EventBus/EventBus.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace StansAssets.Foundation.Patterns
+{
+    /// <summary>
+    /// Basic implementation of the <see cref="IEventBus"/>.
+    /// </summary>
+    public class EventBus : IEventBus
```

Comment:
Any point why we didn't make this class `sealed`?
We have nothing virtual here. Please, consider making this class `sealed`